charts/sysdig-deploy/README.md

```diff
@@ -123,6 +123,16 @@ helm repo update
 helm install -n sysdig-agent sysdig sysdig/sysdig-deploy -f values.new.yaml
 ```
 
+### Differences between `sysdig` and `sysdig-deploy`
+
+When moving from the old chart to new, there will be some differences for the agent and node-analyzer components. The majority of these differences are in metadata names and labels.
```

Comment:
There are several differences in the agent and node-analyzer components in the new chart compared to the old one. The majority of the differences are in the metadata names and labels.